website/src/components/Install/index.tsx

```diff
@@ -38,17 +42,41 @@ export const Install: React.FunctionComponent = () => {
     }, [copied])
 
     return (
-        <div className="bg-white p-5">
-            <h4>
-                Install Sourcegraph locally
-                <span onClick={copy} role="button">
-                    <img src="/copy-text-icon.svg" alt="copy script to clipboard" className="icon-inline ml-4" />
-                </span>
-            </h4>
-
-            <code>
-                <small className={copied ? styles.flashBackground : ''}>{installText}</small>
-            </code>
+        <div className={classNames(styles.windowUI, 'bg-white overflow-hidden')}>
+            <div className={classNames(styles.windowActions, 'd-flex align-items-center px-4')}>
+                {new Array(3).fill(null).map((a, i) => (
+                    <span key={i} className={classNames(styles.windowAction, 'bg-white rounded-circle mr-2')} />
+                ))}
+            </div>
+
+            <div className="p-5">
+                <h3 className="font-weight-bold">
+                    Install Sourcegraph locally
+                    <span
+                        onClick={copy}
+                        role="button"
+                        className={classNames(styles.icon, 'icon-inline ml-4 align-text-top')}
+                    >
+                        <img src={copyIcon} draggable={false} className="w-100" />
+                    </span>
+                </h3>
+
+                <code className="d-block my-4 pr-5 text-lg">
+                    <small className={copied ? classNames(styles.flashBackground, 'text-break') : 'text-break'}>
+                        {installText}
+                    </small>
+                </code>
+
+                <a
+                    className="d-inline-block text-lg"
+                    href="https://docs.sourcegraph.com"
+                    data-button-style={buttonStyle.arrowWithText}
+                    data-button-location={buttonLocation.trySourcegraph}
+                    data-button-type="cta"
```

Comment:
I missed the info around our button tracking. Would you mind giving me some background info @bretthayes or pointing me to our documentation? Is this data getting captured by Google Analytics somewhere? @bretthayes

Reply:
Yeah we're using this for analytics and they're used to track CTA clicks and where they live. It was implemented in #5108

Reply:
Thank you for the background! 🙌